indexer/lib/fwss-handlers.js

```diff
@@ -64,18 +65,40 @@ export async function handleFWSSDataSetCreated(
 /**
  * Handle Filecoin Warm Storage Service service termination
  *
- * @param {Env} env
- * @param {any} payload
+ * @param {{
+ *   DEFAULT_LOCKUP_PERIOD_DAYS?: number
+ *   GENESIS_BLOCK_TIMESTAMP?: number
+ *   DB: D1Database
+ * }} env
+ * @param {object} payload
+ * @param {string} payload.data_set_id
+ * @param {string} payload.block_number
  * @throws {Error}
  */
 export async function handleFWSSServiceTerminated(env, payload) {
+  const DEFAULT_LOCKUP_PERIOD_DAYS = env.DEFAULT_LOCKUP_PERIOD_DAYS || 30 // 30 days default
+  const GENESIS_BLOCK_TIMESTAMP = env.GENESIS_BLOCK_TIMESTAMP || 1667326380 // Calibration genesis as fallback
```

Comment:
This env value must always be configured, and the service mustn't startup if it isn't. A fallback is dangerous here

Reply:
I added asserts to indexer `fetch` method

https://github.com/filbeam/worker/pull/324/commits/f3084d58775c0b46c404e7a984f7efda7a209b59